### DISCUSSION TITLE
Not being able to install homebrew on M1 Macbook with macOS Big Sur version 11.4

### DISCUSSION THREAD

**Discussion by alextalmazan** [Homebrew/discussions#1661] [Getting started]
This happens every time I try to install it:

==> Tapping homebrew/core
remote: Enumerating objects: 971821, done.
remote: Counting objects: 100% (206/206), done.
remote: Compressing objects: 100% (106/106), done.
error: 3703 bytes of body are still expected.36 MiB | 4.34 MiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed
Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master

**Comment by danielnachun** [comment]
Do you have any sort of firewall or other software that filters network traffic installed?  Are you on any sort of enterprise network where network traffic is filtered further upstream than your computer?  These are the most common scenarios that cause issues with cloning Homebrew.

**Comment by carlocab** [comment]
See #666 for a related discussion/various solutions.
